src/models/callercalleemodel.cpp

```diff
@@ -47,6 +47,8 @@ QVariant CallerCalleeModel::headerCell(int column, int role) const
             return tr("Symbol");
         case Binary:
             return tr("Binary");
+        case IsInline:
```

Comment:
I don't think it's worthwhile to add this detail to the caller/callee model as a column. just showing it in a tooltip should be fine?

or can you at least make sure that this column is hidden by default?